drivers/tmp006/include/tmp006.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2014 PHYTEC Messtechnik GmbH
+ *               2017 HAW Hamburg
```

Comment:
Yes, you're right - I'll move it back for the time being. But have a look at my proposal in #6923, to get the general idea I'm aiming for.